cranelift/interpreter/src/step.rs

```diff
@@ -522,7 +524,7 @@ where
             let load_ty = inst_context.controlling_type().unwrap();
             let slot = inst.stack_slot().unwrap();
             let offset = sum(imm(), args()?)? as u64;
-            let mem_flags = MemFlags::trusted();
+            let mem_flags = MemFlags::new();
```

Comment:
Me and @jan-justin were discussing this on Zulip, and we were unsure if unaligned `stack_{load,store}`s allowed. By coincidence the fuzzer found a similar issue in #5922 so we will probably get our answer there!